app/controllers/courses_controller.rb

```diff
@@ -17,7 +17,8 @@ def index
       @ransack_courses = Course.published.approved.ransack(params[:courses_search], search_key: :courses_search)
       # @courses = @ransack_courses.result.includes(:user)
 
-      @pagy, @courses = pagy(@ransack_courses.result.includes(:user))
+      @pagy, @courses = pagy(@ransack_courses.result.includes(:user, :course_tags, :course_tags => :tag))
```

Comment:
It's better to use the new symbol key notation `course_tags: :tag`

Reply:
and same thing a few times below....

Reply:
fixed